linode/manager
# feat: [M3-7158] - Update NodeJS naming to Node.js for Marketplace

## Description 📝
Updates any reference to `NodeJS` in Marketplace to `Node.js`.

## Changes  🔄
List any change relevant to the reviewer.
- Text copy in updated to include punctuation and lower case for `js`.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-10-10 at 9 53 25 AM](https://github.com/user-attachments/assets/f616ab62-aee9-446c-b99f-150a2d55535d) | ![Screenshot 2024-10-10 at 9 53 09 AM](https://github.com/user-attachments/assets/6c871ed8-b66f-4cbd-8321-78452033dd94) |
| ![Screenshot 2024-10-10 at 9 53 51 AM](https://github.com/user-attachments/assets/f8ec1dcb-70d4-4820-bad3-e9212b92b120) | ![Screenshot 2024-10-10 at 9 53 46 AM](https://github.com/user-attachments/assets/5d0a6009-20c1-478e-a22c-a32830f51f35) |

## How to test 🧪

### Verification steps
(How to verify changes)
- Visit the **Marketplace** tab and inspect the text copy found by clicking the info icon `(i)` under the following Apps.
  - NodeJS
  - OpenLiteSpeed NodeJS
- Verify that all text copy references of `NodeJS` have been replaced with `Node.js` within the `AppDetailDrawer`.

## As an Author I have considered 🤔

*Check all that apply*
